src/AspNet.Security.OAuth.VkId/VkIdAuthenticationErrors.cs

```diff
@@ -0,0 +1,19 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+*/
+
+namespace AspNet.Security.OAuth.VkId;
+
+public static class VkIdAuthenticationErrors
```

Comment:
There's no need to expose these as constants - they can just be specified in the code where they're used.